refaster-runner/src/main/java/tech/picnic/errorprone/refaster/runner/CodeTransformers.java

```diff
@@ -80,7 +80,7 @@ private static Optional<String> getRefasterRuleName(ResourceInfo resource) {
   @SuppressWarnings({
     "java:S1166" /* The caught exception's stack traces are not relevant. */,
     "java:S2147" /* The `catch` clauses have distinct comments. */,
-    "TrailingComment"
+    "key-to-resolve-AnnotationUseStyle-and-TrailingComment-check-conflict"
```

Comment:
👀 , I don't get this.

Reply:
The `AnnotationUseStyle` and `TrailingComment` rules are in conflict, and the only fix I found was to introduce another suppression constant. (And I don't want to add yet another comment spelling that out; too verbose.)

Reply:
Where is this constant now defined? 🤔 I still don't get it 😬.